### DISCUSSION TITLE
Tensorflow training

### DISCUSSION THREAD

**Discussion by kgoderis** [deepjavalibrary/djl#3065] [Q&A]
The documentation op djl.ai mentions that learning is not available with the Tensorflow engine. Why is that? and, or, is this still relevant, or is the documentation outdated ?

**Comment by frankfliu** [comment]
To supporting training in TensorFlow, we have to implement GradientCollector for TensorFlow and a couple of operators. The demand of training is low, most of users train the model in Python and run inference with DJL.
